feature/settings/src/main/res/values/strings.xml

```diff
@@ -55,14 +55,14 @@
     <string name="flash_mode_description_off">Flash is Off</string>
     <string name="flash_mode_description_llb">Using Low Light Boost</string>
 
-    <!-- Capture mode setting strings -->
-    <string name="capture_mode_title">Set Capture Mode</string>
+    <!-- Stream Config mode setting strings -->
+    <string name="stream_config_title">Set Stream Config</string>
```

Comment:
nit: Maybe refer to it as "Stream Configuration" in the UI. You can leave it as "Config" in code.